rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -1810,6 +1810,141 @@ public void rskTxWaitingForSignature_uses_updateCollection_rskTxHash_before_rski
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test
+    public void rskTxWaitingForSignature_postRSKIP326_emitNewPegoutConfirmedEvent() throws IOException {
```

Comment:
```suggestion
    public void testNewPegoutConfirmedEventIsEmittedPostRSKIP326() throws IOException {
```
We Can Make The Names More Descriptive With CamelCases.. What Do You Think?

Reply:
I think we'll reach a consensus once the core team shares their coding practices document and we can all agree with a naming convention

Reply:
I think the format `methodName_params_expectedResult` is more descriptive. But anyways, the core team is working on the naming convention.